api/app/Providers/AppServiceProvider.php

```diff
@@ -26,5 +28,12 @@ public function boot()
             'personalExperience' => \App\Models\PersonalExperience::class,
             'workExperience' => \App\Models\WorkExperience::class,
         ]);
+        DB::listen(function($query) {
```

Comment:
Oops, don't leave debugging code live.  I use this a lot though... maybe leave it in but commented out?  With a comment to enable it for debugging?